src/Servers/Kestrel/samples/SampleApp/Startup.cs

```diff
@@ -72,8 +71,8 @@ public static Task Main(string[] args)
                 .ConfigureAppConfiguration((hostingContext, config) =>
                 {
                     var env = hostingContext.HostingEnvironment;
-                    config.AddJsonFile("appsettings.json", optional: true)
-                          .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true);
+                    config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                          .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true, reloadOnChange: true);
                 })
                 .UseKestrel((context, options) =>
```

Comment:
Can you show setting kestrel's reloadOnChange option, even it it's to the default?